### ISSUE TITLE
Пропадает главное окно адт (становится не активным) при закрытии второго дочернего окна

### ISSUE THREAD

**Issue by romal2000** [ramer/ADTools#4]
### Environment

- Scope: ADTools, ADToolsSIP, ADToolsTelegramBot, ...
- Version: 0.xx.x | master
- OS: Windows 10 | Windows 7 | etc
- Machine: i3, 4 GB RAM

### Steps to reproduce

Try to narrow down your scenario to a minimal working/failing example.
Start with deleting steps not relevant to the issue and observing the result:
is the problem still there? Repeat until you get the most straightforward
sequence of steps to reproduce the problem without any noise surrounding it.

1. Open this
2. Press that
3. Click here and there

You can attach screenshots.

### Expected behavior

What should happen?

### Actual behavior

What’s happen now?

If it will make the issue more illustrative, attach screenshots or GIF’s of
what’s going on.

### How to fix 

If you have an idea on how the bug can be fixed, provide a short
description.

### Rationale

Why should we refactor? Will it make the code more DRY, KISS, YAGNI?

### Affected modules and packages

Provide a description of scope the refactoring would touch.

Which backward compatibility would be dropped.

### Rationale

What we have now, what we want then? How the implementation would
make the product better for end-users?

### Prerequisites

Refer to issues by their #numbers that should be completed before
the development of this issue would make sense

### Acceptance criteria

Provide some steps that an end-user would be able to perform when
the feature could be considered as implemented. E.g.:

- [ ] I can click a new Boo menu item
- [ ] A dialog appears where I can enter Foo value
- [ ] Clicking OK changes all nodes’ labels to the Foo value

### Out of scope

Explicitly list functionality that is not intended to be done with this story.

### How to implement

If you have an idea on how to implement it, provide a short
description.

### How to test

How can we test what’s implemented?
